org.deluge_torrent.deluge.yaml

```diff
@@ -0,0 +1,456 @@
+# flatpak-builder --force-clean build-dir org.deluge_torrent.deluge.yaml 
+# flatpak-builder --user --install --force-clean build-dir org.deluge_torrent.deluge.yaml
+# flatpak run --command=bash org.deluge_torrent.deluge ;#testing
+# /app/bin/deluge-gtk -L debug
+
+
+app-id: org.deluge_torrent.deluge
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: deluge-gtk
+#rename-icon: deluge
+#copy-icon: true
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.gnome.SessionManager
+  - --own-name=org.kde.StatusNotifierItem-2-2
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=xdg-download
+  - --env=PATH=/app/bin:/app/runners/bin:/usr/bin
+
+modules:
+
+- name: boost
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2
+    sha256: 953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb
+  build-commands:
+  - ./bootstrap.sh --prefix="${FLATPAK_DEST}" --with-python=python3 
```

Comment:
You should build only what you need from boost.

Reply:
https://dev.deluge-torrent.org/wiki/Installing/Source, seeing the build instructions,  I could not find specific part of partial build of boost modules for deluge. 

```
If building libtorrent:
boost >= 1.34.1
```